### PR TITLE
Linux kernel upgrade and mesa/xorg drivers for DB845c

### PR DESCRIPTION
**Pull request by alimon**

### CODE REVIEW

File: recipes-kernel/linux/linux-qcom-bootimg.inc [hunk 1]
```diff
@@ -10,6 +10,8 @@ BOOT_IMAGE_BASE_NAME = "boot-${KERNEL_IMAGE_NAME}"
 BOOT_IMAGE_SYMLINK_NAME = "boot-${KERNEL_IMAGE_LINK_NAME}"
 SD_BOOT_IMAGE_BASE_NAME = "boot-sd${KERNEL_IMAGE_NAME}" 
 SD_BOOT_IMAGE_SYMLINK_NAME = "boot-sd-${KERNEL_IMAGE_LINK_NAME}"
+KERNEL_CMDLINE_dragonboard-845c = "root=/dev/${1} rw rootwait console=${ttydev},${baudrate}n8 clk_ignore_unused pd_ignore_unused"
```
Comment (ndechesne) [inline recipes-kernel/linux/linux-qcom-bootimg.inc]: KERNEL_CMDLINE_append_dragonboard-845 instead?

File: recipes-graphics/mesa/mesa_%.bbappend [hunk 2]
```diff
@@ -17,3 +18,8 @@ PACKAGECONFIG_append_apq8096 = " \
     gallium \
     ${@bb.utils.contains('DISTRO_FEATURES', 'x11', 'xa', '', d)} \
 "
+
```
Comment (ndechesne) [inline recipes-graphics/mesa/mesa_%.bbappend]: we need to update xserver-xorg_%.bbappend as well.

File: recipes-kernel/linux/linux-qcom-bootimg.inc (hunk 1 above)
Comment (alimon) [reply recipes-kernel/linux/linux-qcom-bootimg.inc]: Right, better to use append, changing...

File: recipes-graphics/mesa/mesa_%.bbappend (hunk 2 above)
Comment (alimon) [reply recipes-graphics/mesa/mesa_%.bbappend]: Ok

**Comment by ndechesne** [comment]
pushed to master as well.

**Pull request merged**